### ISSUE TITLE
[Doc]: Stylesheet reference is too small in new website layout

### ISSUE THREAD

**Issue by timhoffm** [matplotlib/matplotlib#21734]
### Documentation Link

https://matplotlib.org/3.5.0/gallery/style_sheets/style_sheets_reference.html

### Problem

One can hardly see any differences between the styles:

![grafik](https://user-images.githubusercontent.com/2836374/143148227-57898056-6363-4c69-8486-cd3de8765a4b.png)

### Suggested improvement

Adapt figure size, maybe needs a change of layout.

### Matplotlib Version

3.5.0

**Comment by QuLogic** [comment]
At least, probably two columns isn't going to work, but then it'll be pretty long. This _may_ be a case where a carousel or something fancy-ish may be better.

**Comment by timhoffm** [comment]
That'd have to be custom made.

A simpler approach: Split of the seaborn styles. They are a special group and pretty similar to each other. Also, they could use a comment.

```
[normal styles]

seaborn styles
-------------------

The following styles have been adopted from seaborn.

.. note::

   The styles are from seaborn X.Y. Seaborn itself has changed its styles in more recent versions, but for stability, we are keeping them as is.

[seaborn]
```

**Comment by jklymak** [comment]
This would look fine to me as 1-column versus 2?

**Comment by dstansby** [comment]
It looks like sphinx gallery forces multiple figures to be shown in two columns within their CSS:
```css
.sphx-glr-multi-img {
  max-width: 42%;
  height: auto;
}
```
https://github.com/sphinx-gallery/sphinx-gallery/blob/813caade88e42673b6eaa848c607a11497c1a809/sphinx_gallery/_static/sg_gallery.css#L173-L176

so the best way to solve this seems to be an upstream request to sphinx gallery to make this configurable somehow.

**Comment by timhoffm** [comment]
Closed by #21837.
